configs/google_com_proxy.v2.yaml

```diff
@@ -26,7 +26,6 @@ static_resources:
   clusters:
   - name: service_google
     connect_timeout: 0.25s
-    type: LOGICAL_DNS
```

Comment:
dns_lookup_family is the field I was suggesting removing, not this one.

Reply:
But it could be as well, right, as there is a default (I fixed the documentation in the protobuf in the doc PR).

Reply:
I think 'type' should be left as the type of cluster is central to understanding Envoy, even if it is the default. Please leave it.